.github/workflows/lint_pr_labels.yml

```
@@ -17,6 +17,9 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
+permissions:
+  pull-requests: write
```

According to https://github.com/deepakputhraya/action-pr-title/pull/26 we can get away with a read only permission. Can you check it out in your fork @jsoref 🙏🏽 ?

```suggestion
  pull-requests: read
```

---

That doesn't appear to be the action you're using here... 

You're using:
https://github.com/decidim/decidim/blob/f69f55316f884ec5d24ddf5a4fcd30aa43594e69/.github/workflows/lint_pr_labels.yml#L28

Which has this code:
https://github.com/jesusvasquez333/verify-pr-label-action/blob/7cb20ea41884bca232a128769bcb65391c24389f/verify_pr_lables.py#L241